### ISSUE TITLE
I am using your code to do the same thing for a png file but I am having trouble.

### ISSUE THREAD

**Issue by delebash** [Rodrigo-NH/UE4HmapsGenerator#2]
Thanks for the awesome code.  I am trying to draw splines on an imported heightmap 16 bit png file.  I know it is not georeferenced like the geotiff file but I have bounding box lng lat and utm conversions so I think I should be able to place the geojson landscape features such as a road on my landscape.  I have made a little video that explains what I have so far and what I need help with if you have the time.

My two projects are here

Web Application:
This allows you to select a real world location and create a heightmap file for Unreal.
https://github.com/delebash/unreal_mapbox_bridge

MapboxPlugin:
This should place the geojson features such as roads on the imported heightmap file.
https://github.com/delebash/UnrealMapboxBridgePlugin

Note the test Unreal Project is for use with Unreal 5.0 Preview 1 not 5.0 Early Access.

Demo Video
https://1drv.ms/v/s!Aq4OgSZRh__uieNLDnjytI0m9hf5WA?e=8VKBOw

Thanks,
Dan

**Comment by Rodrigo-NH** [comment]
Hello Dan,
Sorry about the delay. It's been a while since the last time I interacted with code & Github (life).
Thanks for the video I enjoyed watching it. Looks like you you're getting a really neat Mapbox plugin there.
Did you sort the issue? (I hope so)

I could not detect what can be wrong after reviewing your video and the importlines logic. It's really simple considering after all what we do is applying the (UTM) offset and convert from meter to UE units (centimeter, the '* 100.00' part)
1- Do your imported landscape retains correct real world aspect ratio and scales? e.g. same X,Y total lengths in UE and real world distances (in UTM projection)? At least that's what I think you're doing after watching the video.
2- It's worth noting that if you just crop/collect your data in geographic coordinate system - GCS, (or any EPSG for the matter, webmaps usually EPSG:3857), then just translating the bounding box (like you did with the latlong.net) and use that as reference to the original data, it will contain distortions probably. Probably It will not be a correct real world representation (a correct UTM representation in the case). Need to reproject the data to guarantee you have a reliable/correct UTM representantion. Point is different EPSGs are not just different units but can also mean different 'shapes' of your data. Example https://gis.stackexchange.com/questions/344641/epsg-3857-vs-utm-coordinate
3- Probably not the case but also worth: If you try importing a UTM geojson actually located in other UTM grid (10T in your example) it will surely not make sense as X coord repeats over different zones. See attached image.

Sorry, maybe you're aware of these things already but this is all I can think trying to spot the issue ATM. And I mean: If its the case your representation is distorted, I dont know by how much, then not sure how much a displacement will exist trying to add data further.
Not sure if useful but I have a blueprint to check UTM coords in UE (but still UE4...). It basically do the inverse thing that importlines.py do, so perhaps a good place to look. https://github.com/Rodrigo-NH/UE4HmapsGenerator/tree/main/blueprints

I think it makes sense leaving this issue open for now.
Thanks!

![image](https://user-images.githubusercontent.com/16311264/164881311-80c4c7e5-b7e7-40a9-9e07-1d4b34550f5d.png)
